src/Employer/Employer.Web/Controllers/ApplicationReviewController.cs

```diff
@@ -28,15 +30,43 @@ public async Task<IActionResult> ApplicationReview(ApplicationReviewRouteModel r
         [HttpPost("", Name = RouteNames.ApplicationReview_Post)]
         public async Task<IActionResult> ApplicationReview(ApplicationReviewEditModel m)
         {
-            if (ModelState.IsValid == false)
+            if (!ModelState.IsValid)
             {
                 var vm = await _orchestrator.GetApplicationReviewViewModelAsync(m);
                 return View(vm);
+            }            
+            var applicationStatusConfirmationViewModel = await _orchestrator.GetApplicationStatusConfirmationViewModelAsync(m);
+            return View("ApplicationStatusConfirmation", applicationStatusConfirmationViewModel);
```

Comment:
This should redirect to a `ApplicationReviewConfirmation_Get` route. We try to stick to Post/Redirect/Get pattern and try not to return views from Posts (except for validation errors).